### ISSUE TITLE
Accessibility: Document language missing, screen reader uses incorrect language

### ISSUE THREAD

**Issue by radinamatic** [learningequality/ka-lite#3350]
Branch: develop

Expected Behavior: When ka-lite language is set to a specific language (in my case Spanish/Castillano), screen reader programs should start reading it in that language.

__WCAG Guidelines: 3.1.1 Language of Page (**Level A**)__

Current Behavior: Tested with JAWS & NVDA, both start in English, because of the missing document language attribute (<html lang="es">).

Steps to reproduce:

1. Download & install language pack
2. Change the default KA lite language
3. Start the screen reader

![doc-lang-missing](https://cloud.githubusercontent.com/assets/1457929/6738797/a5873506-ce74-11e4-921d-32d17e06a2d7.png)

**Comment by radinamatic** [comment]
Fixed in #3527
